tar_scm.py

```diff
@@ -1259,7 +1259,13 @@ def singletask(use_obs_scm, args):
 
     repodir = None
     # construct repodir (the parent directory of the checkout)
-    if repocachedir and os.path.isdir(os.path.join(repocachedir, 'repo')):
+    if repocachedir:
+        # construct subdirs on very first run
+        if not os.path.isdir(os.path.join(repocachedir, 'repo')):
+            os.mkdir(os.path.isdir(os.path.join(repocachedir, 'repo')))
```

Comment:
This looks wrong to me - presumably you don't want the `isdir` inside the `mkdir` call.